packages/cli/src/commands/setup/auth/auth.js

```diff
@@ -324,7 +332,7 @@ export const builder = (yargs) => {
 export const handler = async (yargs) => {
   const { provider, rwVersion } = yargs
   let force = yargs.force
-  let webAuthn = false
+  let includeWebAuthn = yargs.webauthn
```

Comment:
Isn't is possible to have this flag in the dbAuth provider file?

Reply:
It is, but as the comments in there say, it’s theoretically possible to use WebAuthn with other providers, so I put it at the top level…